lib/hooks/api/useContributorList.ts

```diff
@@ -2,15 +2,18 @@ import useSWR, { Fetcher } from "swr";
 import { useState } from "react";
 import { publicApiFetcher } from "lib/utils/public-api-fetcher";
 
-export function convertToContributors(rawContributors: DBListContributor[] = []): DbPRContributor[] {
+export function convertToContributors(
+  rawContributors: DBListContributor[] = [],
+  oscrEnabled: boolean
+): DbPRContributor[] {
   const contributors = rawContributors
     ? rawContributors.map((contributor) => {
         return {
           author_login: contributor.login,
           username: contributor.username,
           updated_at: contributor.created_at,
           user_id: contributor.id,
-          oscr: contributor.oscr,
+          oscr: oscrEnabled ? contributor.oscr : undefined,
```

Comment:
The OSCR rating may be undefined because the user does not have access to it, or on some pages, the data is unavailable, e.g. contributors table on the explore page.